### PR TITLE
DCOS-43353: add style utility typography components

### PR DESCRIPTION
**Pull request by mperrotti**
These components are:
Text
HeadingText1
HeadingText2
HeadingText3
InteractiveText
SmallText
CaptionText
DangerText
WarningText
SuccessText
MonospaceText

See the READMEs in this PR for more info

Many of these stories make use of Storybook's "Knobs" panel

## Screenshots
Too many to capture. Check out the Storybook stories:
Screen Shot 2019-07-04 at 1 34 47 AM

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
3 snapshots are failing.

**Comment by GeorgiSTodorov** [comment]
Other than that, it looks good.

File: packages/styleUtils/typography/components/SmallText.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Text from "./Text";
+import { SharedTextProps } from "../textTypes";
+import { FontWeights } from "../../../shared/styles/styleUtils/typography/weight";
+import { themeTextColorPrimary } from "../../../design-tokens/build/js/designTokens";
+// import { textColorPrimary } from "../../../design-tokens/build/js/designTokens";
```
Comment (GeorgiSTodorov) [inline packages/styleUtils/typography/components/SmallText.tsx]: Forgotten comment.

**Comment by pierrebeitz** [comment]
rebased, removed the basework-commit and used the new `BreakpointConfig`-helper.

**Pull request merged**